### DISCUSSION TITLE
I want to Connect to mongo DB for some tables but it is giving "Database connection [mongodb] not configured.", exception: "InvalidArgumentException",…} error

### DISCUSSION THREAD

**Discussion by iamabhilash-kumar** [spatie/laravel-multitenancy#528] [General]
Hi,

i am using this package for long and loving it a lot. somehow due to some use cases i want to connect to mongo db for some tables . when i am using mongodb connection it is giving me below error.

`"Database connection [mongodb] not configured.", exception: "InvalidArgumentException`

Here is my connection in database.php
```php
'mongodb' => [
        'driver' => 'mongodb',
        'dsn' => env('DB_URI', 'mongodb://localhost'),
        'database' => 'tenant1_db',
    ],
```

And this is how i am switching the db

```php
$result = DB::connection('mongodb')->table('webforms')->create($data);
```

**Comment by masterix21** [comment]
The issue seems to be related to a misconfigured mongodb connection. Could you take a look at your database config?

**Reply by iamabhilash-kumar** [reply]
Thanks, @masterix21 the problem is solved now. it was due to some mis configuration.
